archunit/src/main/java/com/tngtech/archunit/library/freeze/ViolationStoreFactory.java

```diff
@@ -15,32 +15,12 @@
  */
 package com.tngtech.archunit.library.freeze;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.util.List;
-import java.util.Properties;
-import java.util.UUID;
-import java.util.regex.Pattern;
-
-import com.google.common.annotations.VisibleForTesting;
-import com.google.common.base.Joiner;
-import com.google.common.base.Splitter;
 import com.tngtech.archunit.ArchConfiguration;
 import com.tngtech.archunit.base.MayResolveTypesViaReflection;
-import com.tngtech.archunit.lang.ArchRule;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 import static com.google.common.base.Preconditions.checkArgument;
```

Comment:
unused, just as the following `checkState`

Reply:
removed